modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,89 @@
+** `tkn task list`
+** `tkn pipeline list`
+** `tkn taskrun list`
+** `tkn pipelinerun list`
+
+The output of these commands is also enhanced to display information without headers using the `--no-headers` flag option.
+
+* You can now start a Pipeline using default parameter values by specifying `--use-param-defaults` flag in the `tkn pipelines start` command.
+* Support for Workspace is now added to `tkn pipeline start` and `tkn task start` commands.
+* A new `clustertriggerbinding` command is now added with sub-commands - `describe`, `delete`, and `list`.
+* You can now directly start a pipeline run using a local or remote `yaml` file.
+* The `describe` sub-command now displays an enhanced and detailed output. With the addition of new fields,  such as `description`, `timeout`, `param description`, and `sidecar status`, the command output now provides more detailed information about a specific `tkn` resource.
+* The `tkn task log` command will now display logs directly if only one task is present in the namespace.
+
+=== Triggers
+* Triggers can now create both `v1alpha1` and `v1beta1` Pipeline resources.
+* Support for new Common Expression Language (CEL) interceptor function - `compareSecret`. This function securely compares strings to secrets in CEL expressions.
+* Support for authentication and authorization at the EventListener Trigger level.
+
+== Deprecated features
+The following items are deprecated or not supported in this release:
+
+* The environment variable `$HOME`, and variable `workingDir` in the Steps specification will be changed in the next release. Currently in a Step container, `HOME` and  `workingDir` are overwritten to `/tekton/home` and `/workspace` respectively.
++
+In a later release, these two fields will not be modified, and will be set to values defined in the container image and Task YAML.
+For this release, use flags `disable-home-env-overwrite` and `disable-working-directory-overwrite` to disable overwrite of the `HOME` and `workingDir` variables.
+
+* The following commands are deprecated and will be removed in the next release:
+** `tkn pipeline create`
+** `tkn task create`
+
+* The `-f` flag with the `tkn resource create` command is now deprecated. It will be removed in the next release.
+
+* The `-t` flag and the `--timeout` flag (with seconds format) for the `tkn clustertask create` command are now deprecated. Only duration timeout format is now supported, for example `1h30s`. These deprecated flags will be removed in the next release.
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* If you are upgrading from an older version of {pipelines-title}, you will need to delete your existing deployments before upgrading to {pipelines-title} version 1.0. To delete an existing deployment, you first need to delete Custom Resources and then uninstall the {pipelines-title} Operator. For more details, see the uninstalling {pipelines-title} section.
```

Comment:
Again, if this is the first release of OpenShift Pipelines, then "If you are upgrading from an older version of OpenShift Pipelines" doesn't sound right.

Reply:
This is the first release for which we are adding documentation. Since earlier releases were DP, no docs were added. This line is intended for those users who want to upgrade from a DP to new TP release